pcsx2/CDVD/CDVD.cpp

```diff
@@ -1222,6 +1222,7 @@ static void cdvdWrite04(u8 rt) { // NCOMMAND
 				Console.WriteLn( Color_Gray, L"CdRead: Reading Sector %07d (%03d Blocks of Size %d) at Speed=%dx",
 					cdvd.SeekToSector, cdvd.nSectors,cdvd.BlockSize,cdvd.Speed);
 
+			cdvd.PwOff = 1 << Irq_DataReady;
```

Comment:
If I use `cdvdSetIrq(1 << Irq_DataReady);` in the N_DVD_READ or N_CD_READ cases it seems to break things.
```
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
(EE pc:80005560) TLB Miss, addr=0x0 [load]
Too many execution errors.  VM execution has been suspended!
```
I guess the same will apply to the N_CD_READ_CDDA and N_CD_READ_XCDDA case. Any ideas on what to try?

About the N_CD_CHG_SPDL_CTRL case - I'm not sure the change I made even makes sense now that I think about it, but yeah, I do know pretty much nothing about CDVD emulation stuff.